### PR TITLE
chore(ui): bug squashing

### LINKED ISSUE #518: The example org name is not actually a valid org name
Just make the "F" lowercase.

image

### LINKED ISSUE #530: Border radius on tooltips should be 4px instead of 6px
currently:
image

---

should be 4px:
image

### LINKED ISSUE #532: Update first-time intro screen to new design
image

image

Need to update this intro placeholder.
- [ ] Replace this placeholder image with the Radicle logo.
- [ ] Add intro paragraph text. "A free and open-source way to host, share, and build software together."

Here's the [Figma design](https://www.figma.com/file/dnYlebm0ehgyJyT3Po9Kcl/Upstream?node-id=13068%3A250) for reference.

### LINKED ISSUE #536: "Settled" should be lowercase
image

### PR DESCRIPTION
**Pull request by juliendonck**
- closes #530 
- closes #532 
- closes #518 
- closes #536

### CODE REVIEW

**Comment by rudolfs** [comment]
@juliendonck is there an easy way to get the pointer rounded?

Screenshot 2020-06-22 at 14 34 09

**Comment by juliendonck** [comment]
> @juliendonck is there an easy way to get all the edges of the pointer rounded?
> 
> Screenshot 2020-06-22 at 14 34 09

@rudolfs no

**Comment by rudolfs** [comment]
Hm, yeah, thought so. I tried to get it right a while ago, but decided that we'll have to re-work the tooltips anyway to support showing them in different directions.

File: ui/Screen/OrgRegistration.svelte [hunk 1]
```diff
@@ -110,7 +110,7 @@
     </Text>
     <Input.Text
       dataCy="input"
-      placeholder="Org name (e.g. Flowerpot)"
+      placeholder="Org name (e.g. flowerpot)"
```
Comment (xla) [inline ui/Screen/OrgRegistration.svelte:113]: :seedling: :seedling: :seedling:

**Comment by juliendonck** [comment]
> Hm, yeah, thought so. I tried to get it right a while ago, but decided that we'll have to re-work the tooltips anyway to support showing them in different directions and gave up.

@rudolfs got it to work

**Pull request merged**